microsoft-365/enterprise/PortalLaunchScheduler.md

```diff
@@ -21,81 +21,116 @@ description: "This article describes how you can launch your portal using the Po
 
 # Launch your portal using the Portal Launch Scheduler
 
-You can launch your portal using the Portal Launch Scheduler by first validating the tenant admin's ability to setup a waves for a new portal. Then the admin can validate a redirection of requests, based on the existence of a user in the active waves.
+A portal is a SharePoint site on your intranet that has a large number of site viewers who consume content on the site. Launching your portal in waves is an important part of ensuring users have a smooth and performant experience accessing a new SharePoint Online portal. 
+
+Launching in waves is a key way to roll-out your portal, as detailed in [Planning your portal launch roll-out plan in SharePoint Online](https://docs.microsoft.com/en-us/microsoft-365/Enterprise/Planportallaunchroll-out?view=o365-worldwide). The Portal Launch Scheduler is designed to help you follow a wave / phased roll-out approach by managing the redirects for the new portal. During each of the waves, you can gather user feedback and monitor performance during each wave of deployment. This has the advantage of slowly introducing the portal, giving you the option to pause and resolve issues before proceeding with the next wave, and ultimately ensuring a positive experience for your users. 
+
+There are two types of redirection: 
+- bidirectional: launch a new modern SharePoint Online portal to replace an existing SharePoint classic or modern portal 
+- temporary page redirection: launch a new modern SharePoint Online portal with no existing SharePoint portal
+
+The portal launch scheduler is only available to launch modern SharePoint Online portals, like communication sites and modern team sites. Launches must be scheduled at least 7 days in advance. The number of waves required is determined by the expected number of users. Before scheduling a portal launch, the [Page Diagnostics for SharePoint tool](https://aka.ms/perftool) must be run to verify that the home page on the portal is healthy. At the end of the portal launch, all users with permissions to the site will be able to access the new site. 
```

Comment:
Need to change this to "...available to launch modern SharePoint Online portals (i.e. communication sites)."